redi-connect-front/src/components/atoms/FormSelect.tsx

```diff
@@ -45,14 +45,14 @@ function FormSelect (props: any) {
       ...provided,
       heihg: '40px',
       padding: '13px',
-      color: state.isSelected ? 'black' : '',
-      backgroundColor: state.isSelected ? '#dadada' : '',
-      '&:hover': {
+      color: state.isFocused ? 'black' : '',
```

Comment:
Why are we changing this?

Reply:
This solves the keystroke problem ... without the change the custom dropdown wouldn't work on keypress but I figured if you reverse it (`isFocused` instead of `isSelected` + `&:active` instead of `&:hover`) it works perfectly fine. 

The `&:active` on line 49 is only needed if you confirm the language with a click instead of a keypress. If you do so you see a blue background `onClick` instead of the wanted grey background.

Reply:
Great!